### ISSUE TITLE
Google Play link is not available

### ISSUE THREAD

**Issue by pcqpcq** [dodyg/AndroidRivers#15]
https://play.google.com/store/apps/details?id=com.silverkeytech.android_rivers

it's broken.

**Comment by dodyg** [comment]
Yeah, I am going to republish in a  month or so once everything gets upgraded. I am dropping support for Android 2.2, etc so we can finally ditch SherlockActionBar and other old stuff.

**Comment by pcqpcq** [comment]
Good, I want to add this to [my collection](https://github.com/pcqpcq/open-source-android-apps), I think it will be better at that time! :+1:
